homeworks/react-basics/vitalikmoroz/src/App.js

```diff
@@ -0,0 +1,53 @@
+import React, {Component} from 'react';
+import Axios from 'axios';
+import './App.css';
+
+class App extends Component {
+	state = {
+		error : false,
+		lectures : []
+	}
+
+	componentDidMount() {
+		this.getLectures();
+	}
+
+	async getLectures() {
+		await Axios.get(`https://raw.githubusercontent.com/kottans/chernivtsi-frontend-2019/master/homeworks/react-basics/data.json`)
+		.then(response => this.setState({
+			lectures : response.data.lectures
+		}))
+		.catch(error => this.setState({
+			error
+		}))
+	}
+
+	renderLectures(lectures) {
```

Comment:
Better move `renderLectures()` to separate file as function component and then render it as 
```
{items && <Lectures/>}
```